readme.md

```diff
@@ -1,96 +1,2323 @@
-# Azure Industrial IoT Platform
+[Home](../../readme.md)
```

Comment:
@dacolgit, I believe that the intent was to change operations manual which is `docs/manual/readme.md`, right ? Seems like the updated content from there has been copy-pasted to `readme.md`. Can you please fix this and apply the changes in `docs/manual/readme.md` ?